Userland/Applications/FileManager/FileUtils.cpp

```diff
@@ -67,4 +68,91 @@ void delete_paths(const Vector<String>& paths, bool should_confirm, GUI::Window*
     }
 }
 
+void create_trash_bin()
+{
+    auto trash_bin_path = String::formatted("{}/.trash", getenv("HOME"));
+    auto trash_bin_files = String::formatted("{}/files", trash_bin_path);
+    auto trash_bin_info = String::formatted("{}/info", trash_bin_path);
+
+    if (Core::File::is_directory(trash_bin_path)) {
+        dbgln("Trash directory exists.");
+    } else {
+        if (mkdir(trash_bin_path.characters(), 0755) < 0) {
+            auto saved_errno = errno;
+            warnln("Failed to create directory {}: {}", trash_bin_path, strerror(saved_errno));
+        }
+    }
+
+    if (!Core::File::is_directory(trash_bin_files)) {
+        if (mkdir(trash_bin_files.characters(), 0755) < 0) {
+            auto saved_errno = errno;
+            warnln("Failed to create directory {}: {}", trash_bin_files, strerror(saved_errno));
+        }
+    }
+
+    if (!Core::File::is_directory(trash_bin_info)) {
+        if (mkdir(trash_bin_info.characters(), 0755) < 0) {
+            auto saved_errno = errno;
+            warnln("Failed to create directory {}: {}", trash_bin_info, strerror(saved_errno));
+        }
+    }
+}
+
+[[maybe_unused]] void move_to_trash(String const& path)
```

Comment:
This function has multiple instances where failure is possible.

Would a return `Boolean` be useful to know whether this function failed?

Reply:
I agree with you, I will do as suggested :)